packages/fxa-settings/src/models/reliers/base-relier.ts

```diff
@@ -172,9 +195,69 @@ export class BaseRelier extends ModelDataProvider implements Relier {
     return false;
   }
   pickResumeTokenInfo() {
-    return {};
+    return {
+      entrypoint: this.entrypoint,
+      entrypointExperiment: this.entrypointExperiment,
+      entrypointVariation: this.entrypointVariation,
+      resetPasswordConfirm: this.resetPasswordConfirm,
+      style: this.style,
+      utmCampaign: this.utmCampaign,
+      utmContent: this.utmContent,
+      utmMedium: this.utmMedium,
+      utmSource: this.utmSource,
+      utmTerm: this.utmTerm,
+    };
+  }
+
+  getResumeToken() {
```

Comment:
I think after I'd [commented](https://github.com/mozilla/fxa/pull/15516#issuecomment-1631070893) we were trying not to port some of this resume token logic you'd circled back and said we probably don't need it but that some values are read in local storage - is there some resume token logic in this PR that we can strip out?

I am guessing if we need it now, we are going to be stripping a lot of this when we finish the reset PW redesign project that uses codes. 🙈

Reply:
Ah, I probably just missed this and the next method. I'll clean it up.